dist/index.js

```diff
@@ -1567,16 +1584,31 @@ function traverse(schema, path, resolve, rootSchema) {
   }
 
   var copy = {};
-
+   copy.description = "";
   if (Array.isArray(schema)) {
     copy = [];
   }
 
   Object.keys(schema).forEach(function (prop) {
+    var schemadescription;
+    //if there is a description we set it otherwise we want an empty string
+    if(schema[prop].description){
+      schemadescription = schema[prop].description;
+    }
+    else {
+      schemadescription = "";
+    }
     if (typeof schema[prop] === 'object' && prop !== 'definitions') {
-      copy[prop] = traverse(schema[prop], path.concat([prop]), resolve, copy);
+    //Creates an html formatted description and creates the empty body tag if needed
+     copy.description += ("<div> " + prop + " (type: " + schema[prop].type + ") : " + schemadescription + " </div> ");
```

Comment:
Consider using an interpolated string here. It can be easier to read when concatenating a bunch of things.
```
<div>${prop} (type: ${schema[prop].type}) : ${schemadescription}</div>
```